pkg/kudoctl/cmd/package_list_plans.go

```diff
@@ -97,7 +97,7 @@ func displayPlanTable(pf *packages.Files, withTasks bool, out io.Writer) {
 }
 
 func sortedPlanNames(pf *packages.Files) []string {
-	planNames := funk.Keys(pf.Operator.Plans).([]string)
+	planNames, _ := funk.Keys(pf.Operator.Plans).([]string)
```

Comment:
This is totally unrelated and shouldn't be in here, but I had to fix it to push because of the pre-commit hook...

Reply:
This is gone now after merge of master